sbin/adduser_course.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+"""
+Use this script to add a user to courses. Any user added to a course
+will be an instructor.
+"""
+
+import argparse
+import json
+from os import path
+import subprocess
+from sqlalchemy import create_engine, MetaData, Table, bindparam, and_
+
+CONFIG_PATH = path.join(path.dirname(path.realpath(__file__)), '..', 'config')
+with open(path.join(CONFIG_PATH, 'database.json')) as open_file:
+    DATABASE_DETAILS = json.load(open_file)
+DATABASE_HOST = DATABASE_DETAILS['database_host']
+DATABASE_USER = DATABASE_DETAILS['database_user']
+DATABASE_PASS = DATABASE_DETAILS['database_password']
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Adds a user to courses')
+
+    parser.add_argument('user_id', help='user_id of the user to create')
+    parser.add_argument('--course', metavar='arg', action='append', nargs=3, help='[SEMESTER] [COURSE] [REGISTRATION_SECTION]')
```

Comment:
Let's make this three positional arguments instead of the flag as the first two are required to even do anything here, and let's just make it only workable with one course at a time. It's a slight degregation of what you could do, but I'm not sure anyone was actually using this to add a user to multiple courses at once anyway.
```
parser.add_argument('semester', type=str)
parser.add_argument('course', type=str)
parser.add_argument('registration_section', type=int, nargs='?', default=None)
```
(add appropriate help messages)

Reply:
Sure! I was also wondering if it was used for adding multiple courses by other users so I didn't change it. Will modify that to one course only.